proprietary/noordwijk-design-tokens/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "version": "1.0.0-alpha.1",
+  "author": "Community for NL Design System",
+  "description": "NL Design System design tokens for Gemeente Noordwijk",
+  "website": "https://noordwijk.nl/",
+  "keywords": [
+    "nl-design-system",
+    "conduction"
+  ],
+  "license": "SEE LICENSE IN LICENSE.md",
+  "name": "@nl-design-system-unstable/noordwijk-design-tokens",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git+ssh",
+    "url": "git@github.com:nl-design-system/themes.git"
+  },
+  "scripts": {
+    "clean": "rimraf -rf dist/",
+    "prebuild": "npm run clean",
+    "watch": "npm-run-all watch:**",
+    "watch:style-dictionary": "chokidar --follow-symlinks --command 'npm run --ignore-scripts build' 'src/**/*.tokens.json'",
+    "build": "npm-run-all build:**",
+    "build:scss": "sass --no-source-map src/:dist/",
+    "build:style-dictionary": "style-dictionary build --config ./style-dictionary.config.js"
+  },
+  "devDependencies": {
+    "@nl-design-system-unstable/theme-toolkit": "workspace:*",
+    "chokidar-cli": "3.0.0",
+    "npm-run-all": "4.1.5",
+    "rimraf": "3.0.2",
+    "style-dictionary": "3.8.0"
+  },
```

Comment:
You can add the font as dependency, instead of relying on the Google CDN. This helps make the theme usable for DigiD audited pages.
https://www.npmjs.com/package/@fontsource/roboto

If you do find in project for `@fontsource` I'm sure you can find examples to copy from.